src/view/com/composer/text-input/TextInput.web.tsx

```diff
@@ -150,16 +150,26 @@ export const TextInput = React.forwardRef(function TextInputImpl(
         attributes: {
           class: modeClass,
         },
-        handlePaste: (_, event) => {
-          const items = event.clipboardData?.items
+        handlePaste: (view, event) => {
+          const clipboardData = event.clipboardData
 
-          if (items === undefined) {
-            return
-          }
+          if (clipboardData) {
```

Comment:
invert condition to adopt "return early" pattern and this to avoid one extra indent : for here, leave quickly when clipboardData is not defined (but IDK if it's possible to have undefined clipboardData on handlePaste event? doubleCheck ^^)